### DISCUSSION TITLE
pyenv and Fabric installation

### DISCUSSION THREAD

**Discussion by clarkebishop** [danielmiessler/Fabric#398] [Q&A]
I was surprised to see that .python_version was set to fabric. I use pyenv to manage Python versions. I see that fabric requires Python > 3.10.

I tried to install fabric with pipx install ., but did not get a clean installation. Should I have used pyenv first to set the Python version?

**Comment by RoeslerM** [comment]
yes, my pyenv is set to 3.12 and that works fine using PIP
